src/lib/evil/unposix/unistd.h

```diff
@@ -1,8 +1,20 @@
-#ifndef UNISTD_H
-#define UNISTD_H
+#ifndef UNPOSIX_UNISTD_H
+#define UNPOSIX_UNISTD_H
 
 #include "unimplemented.h"
 
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#include <corecrt.h>
+#include <BaseTsd.h>
+
+#include <windows.h>
+#include <corecrt_io.h>
+#include <process.h>
```

Comment:
As it is still a PR I think we can already work on #33 and add a `#undef WIN32_LEAN_AND_MEAN`